os/board/rtl8730e/src/component/bluetooth/example/ble_central/ble_tizenrt_client.c

```diff
@@ -183,6 +183,26 @@ void scan_stop_cb(void *arg)
 	} while(new_state.gap_scan_state != GAP_SCAN_STATE_IDLE);
 }
 
+trble_result_e rtw_ble_client_set_scan(uint16_t scan_interval, uint16_t scan_window, trble_scan_type scan_type)
+{
+	rtk_bt_le_scan_param_t gap_scan_param;
+	gap_scan_param.type = scan_type;
+	gap_scan_param.interval = scan_interval;
+	gap_scan_param.window = scan_window;
+	gap_scan_param.own_addr_type = RTK_BT_LE_ADDR_TYPE_PUBLIC;
+	gap_scan_param.filter_policy = RTK_BT_LE_SCAN_FILTER_ALLOW_ALL;
+	gap_scan_param.duplicate_opt = RTK_BT_LE_SCAN_DUPLICATE_DISABLE;
+	if(RTK_BT_OK != rtk_bt_le_gap_set_scan_param(&gap_scan_param))
```

Comment:
Could you correct the coding style
if ( and } else {

Reply:
has amended accordingly